doc/conf.py

```diff
@@ -23,5 +23,5 @@
 
 intersphinx_mapping = {
     "https://docs.python.org/3": None,
-    "urwid": ("https://urwid.org/", None),
+    "urwid": ("https://urwid.readthedocs.io/en/latest/", None),
```

Comment:
When I follow the readthedocs link it shows me version 1.3.1, but the urwid.org link shows version 2.1.2. The github page for urwid links to urwid.org. Have they simply not updated their page on readthedocs in a while?

Reply:
You're right. It looks like https://urwid.org/ is still the official source. That had TLS issues yesterday, which made me go look for an alternative. Removed.